pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings=
+    ignore::DeprecationWarning
```

Comment:
ideally we shouldn't suppress warnings. Instead, we should just change the code so that they are not emitted any more ...

Reply:
I agree, but it is caused at the mne.io.Raw function call which is gonna be hard to avoid. But I've found a way to ignore this warning more focally. The commit is following

Reply:
hmm ... but I don't have this problem on `mne`. What exactly can I reproduce the warning locally? Is it also a problem on the development version?

Reply:
Oh, sorry. It happens only for mne 0.15 and after update to mne 0.16 it disappears. I thought I had the latest version of it but I didn't. I will remove the warnings suppression part from the pull request.